themes/default/content/docs/intro/compare-pulumi-to/terraform.md

```diff
@@ -1,13 +1,15 @@
 ---
-title: Pulumi vs. Terraform
+title: Compare Pulumi to Terraform
```

Comment:
We are at least going to want to leave this page as-is because we rank really well for the query "pulumi vs terraform" and we are going to want to maintain that ranking.

If we are really motivated to change the this to be "compare" instead of "vs" then we should keep the "vs" version of the page around so it will continue to rank well SEO wise. That means we would maintain two versions of the same page, but it would be safer SEO wise.

Reply:
@zchase i can leave all the page titles/urls as vs. is it okay to change the category name to `Compare Pulumi to...` or will that have a negative effect?

Reply:
@zchase i changed them all back to keep their original titles and page names, minus the ones that were using underscores in the URLs

do you think its still okay that the parent name changes to the compare phrase? or will that also have a negative effect?

cc/ @infin8x

Reply:
The page title will still need to be "Pulumi vs X" but the H1 tag could use the "compare" phrasing. What is important SEO wise is that our keywords are in the Title and URL (as well as the content). The H1 is more a "vanity" thing and varying the Title and H1 should have very little impact to SEO.

Changing the parent page should be fine as the SEO juice is really on a per-page basis.